src/components/ChatEntry.js

```diff
@@ -1,22 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({id, sender, body, timeStamp, liked, updateLikes }) => {
+  const handleLike = () => {
+    updateLikes(id);
+  };
 
-const ChatEntry = (props) => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className="chat-entry">
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <p className="entry-time">
+          <TimeStamp time={timeStamp} />
+        </p>
+        <button className={liked ? 'like liked' : 'like'} onClick={handleLike}>
```

Comment:
Interestingly, there doesn't seem to be a style rule applied to `like` or `liked`!